docs/extending_filer.rst

```diff
@@ -308,3 +308,29 @@ Set ``FILER_IMAGE_MODEL`` to the path of your custom model:
 .. code-block:: python
 
     FILER_IMAGE_MODEL = 'myapp.CustomImage'
+
+
+Adding custom file upload constraints
+.....................................
+
+Set ``FILER_FILE_CONSTRAINTS`` to project setting file with path custom method which does the validation.
```

Comment:
Do you mean `settings.py`? If so please use that instead of `setting file`. I'm not sure what "with path custom method" means... seems incomplete... do you mean "with path and custom method"? Or do you mean "with path of custom method?" Or something else?